docs/administration/Configuration.md

```diff
@@ -42,462 +42,2042 @@ ADMIN SET FRONTEND CONFIG ("key" = "value");
+##### catalog_try_lock_timeout_ms
+
+- **Unit**: ms
+- **Default**: 5000
+- **Description**: The timeout duration to obtain the global lock.
+
+##### edit_log_roll_num
+
+- **Unit**: -
+- **Default**: 50000
+- **Description**: The maximum number of metadata log entries that can be written before a log file is created for these log entries. This parameter is used to control the size of log files. The new log file is written to the BDBJE database.
+
+##### ignore_unknown_log_id
+
+- **Unit**: -
+- **Default**: FALSE
+- **Description**: Whether to ignore an unknown log ID. When an FE is rolled back, the BEs of the earlier version may be unable to recognize some log IDs. If the value is `TRUE`, the FE ignores unknown log IDs. If the value is `FALSE`, the FE exits.
+
+##### ignore_materialized_view_error
+
+- **Unit**: -
+- **Default**: FALSE
+- **Description**: Whether FE ignores the metadata exception caused by materialized view errors. If FE fails to start due to the metadata exception caused by materialized view errors, you can set this parameter to `true` to allow FE to ignore the exception. This parameter is supported from v2.5.10 onwards.
+
+##### ignore_meta_check
+
+- **Unit**: -
+- **Default**: FALSE
+- **Description**: Whether non-leader FEs ignore the metadata gap from the leader FE. If the value is TRUE, non-leader FEs ignore the metadata gap from the leader FE and continue providing data reading services. This parameter ensures continuous data reading services even when you stop the leader FE for a long period of time. If the value is FALSE, non-leader FEs do not ignore the metadata gap from the leader FE and stop providing data reading services.
+
+##### meta_delay_toleration_second
+
+- **Unit**: s
+- **Default**: 300
+- **Description**: The maximum duration by which the metadata on the follower and observer FEs can lag behind that on the leader FE. Unit: seconds. If this duration is exceeded, the non-leader FE stops providing services.
+
+##### drop_backend_after_decommission
+
+- **Unit**: -
+- **Default**: TRUE
+- **Description**: Whether to delete a BE after the BE is decommissioned. `TRUE` indicates that the BE is deleted immediately after it is decommissioned. `FALSE` indicates that the BE is not deleted after it is decommissioned.
+
+##### enable_collect_query_detail_info
+
+- **Unit**: -
+- **Default**: FALSE
+- **Description**: Whether to view the profile of a query. If this parameter is set to `TRUE`, the system collects the profile of the query. If this parameter is set to `FALSE`, the system does not collect the profile of the query.
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/05fdeda3-8a0d-4d4e-a04d-74318776662f)
I think it's better to use "collect" compared to "view", 'cause "collect" is an action done by the system while "view" is an action done by the user and clearly here it refers an action done by the system.